Panels/Panel.js

```diff
@@ -260,6 +260,19 @@ const PanelDecorator = hoc({defaultElement: `.${componentCss.body} *`}, (config,
 			continue5WayHold: true,
 			defaultElement,
 			enterTo: 'last-focused',
+			lastFocusedPersist: (node, all) => {
+				const filtered = all.filter(element => element && element.dataset && !element.dataset.spotlightIgnoreRestore);
```

Comment:
How about making the expression shorter with the optional chaining operator?
```suggestion
				const filtered = all.filter(element => element && !element.dataset?.spotlightIgnoreRestore);
```

Reply:
I fixed